frontend/public/components/utils/icon.tsx

```diff
@@ -0,0 +1,5 @@
+import * as _ from 'lodash-es';
+
+export const normalizeIconClass = (iconClass) => {
```

Comment:
Instead of moving this function, I'd rather generalize the other module like in https://github.com/openshift/console/pull/534/files#diff-a02bf0306cfe8689dae7bb19239167a3

Reply:
Ok, I'll readdress after #534 is merged